### PR TITLE
ZORA implementation

### PR DESCRIPTION
**Pull request by Andersmb**
This PR features a ZORA implementation with the possibility of constructing the ZORA potential with `V_nuc`, `J`, and `V_xc`.

### CODE REVIEW

**Comment by stigrj** [comment]
The Betzy example managed to creep back in...

**Comment by Andersmb** [comment]
There are two betzy files in that dir differing only in the case. Since my Mac OS is not case sensitive, it messes up my git 😠 What should I do?

**Comment by stigrj** [comment]
How about `rm <the-file-you-dont-want>`? :slightly_smiling_face:

**Comment by Andersmb** [comment]
Some tests fail linux (tested on saga), but passes on my Mac. Perhaps I need to rerun the reference calculations? (but weird that they pass on one system but fail on another?)

File: src/qmoperators/two_electron/FockBuilder.cpp [hunk 1]
```diff
@@ -83,6 +83,7 @@ void FockBuilder::setup(double prec) {
     this->perturbation().setup(prec);
 
     if (isZora()) {
+        println(0, "isZora!!");
```
Comment (Andersmb) [inline src/qmoperators/two_electron/FockBuilder.cpp:86]: Do you want the `println`  in?

**Comment by Andersmb** [comment]
Did the tests fail due to my sloppy code? 😸

File: src/qmoperators/two_electron/FockBuilder.cpp (hunk 1 above)
Comment (stigrj) [reply src/qmoperators/two_electron/FockBuilder.cpp:86]: oops

**Comment by Andersmb** [comment]
Everything works as expected for me. The new relativity string options in `WaveFunction.relativity` reproduce the explicitly set `ZORA.include_` options.

**Review by stigrj — APPROVED** [review]

**Comment by stigrj** [comment]
@Andersmb you should continue from a fresh `master` when this is merged

**Pull request merged**